src/models/event.ts

```diff
@@ -1003,6 +1039,14 @@ export class MatrixEvent extends EventEmitter {
         return this.getType() === "m.room.redaction";
     }
 
+    public isVisibilityChange(): boolean {
+        const relation = this.getRelation();
+        if (!relation) {
+            return false;
+        }
+        return relation.rel_type == "m.visibility";
```

Comment:
This cannot and should not be using the `m.*` namespace yet. Please use your unstable prefix, ideally through the `UnstableValue` class.